index.d.ts

```diff
@@ -92,19 +60,55 @@ export default class Podlet {
 
     defaults(context: any): any;
 
-    view(
+    view<T = { [key: string]: unknown }>(
         template: (
-            incoming: HttpIncoming,
+            incoming: HttpIncoming<T>,
             fragment: string,
             ...args: unknown[]
         ) => string,
     ): void;
 
-    render(
-        incoming: HttpIncoming,
+    render<T = { [key: string]: unknown }>(
+        incoming: HttpIncoming<T>,
         fragment: string,
         ...args: unknown[]
     ): string;
 
     process(incoming: HttpIncoming): Promise<HttpIncoming>;
 }
+
+declare namespace Podlet {
+    export type AbsLogger = {
```

Comment:
You should be able to replace this with an import now that we have types in the abslog module
https://github.com/trygve-lie/abslog/pull/255